static/app/views/starfish/views/webServiceView/spanGroupBar.tsx

```diff
@@ -62,19 +73,50 @@ export function SpanGroupBar() {
               spanModule === 'db'
                 ? `/${routingContext.baseURL}/performance/database/`
                 : '';
+            function handleModulePin() {
+              if (spanModule === pinnedModule) {
+                setPinnedModule(null);
+                onHover(null);
+              } else {
+                setPinnedModule(spanModule as string);
```

Comment:
```suggestion
                setPinnedModule(spanModule);
```
I think you can leave out the type assertion here, since spanModule seems to already be a string?